packages/blocks/src/api/parser/index.js

```diff
@@ -302,10 +274,10 @@ export function parseRawBlock( rawBlock, options ) {
  * @see
  * https://developer.wordpress.org/block-editor/packages/packages-block-serialization-default-parser/
  *
- * @param {string}       content The post content.
- * @param {ParseOptions} options Extra options for handling block parsing.
+ * @param {string}            content The post content.
+ * @param {BlockParseOptions} options Extra options for handling block parsing.
```

Comment:
`options` is optional

```suggestion
 * @param {string}            content   The post content.
 * @param {BlockParseOptions} [options] Extra options for handling block parsing.
```